dirs/dirs.go

```diff
@@ -88,6 +89,7 @@ func SetRootDir(rootdir string) {
 	SnapdSocket = filepath.Join(rootdir, "/run/snapd.socket")
 
 	SnapBinariesDir = filepath.Join(SnapAppsDir, "bin")
+	SnapDesktopDir = filepath.Join(rootdir, "/usr/share/applications")
```

Comment:
I guess we will need a /apps/desktop style dir and add it to the search path of the xdg desktop search path. But not something I want to think about just yet :/